### ISSUE TITLE
Platypus: Missing level-one heading

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#470]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page is missing level-one heading.

---

https://dequeuniversity.com/rules/axe/4.3/page-has-heading-one?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Sighted Keyboard Users, Blind, Low Vision, Deafblind
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A

**Comment by micpap25** [comment]
Hi, I believe this can be closed. The level-one heading looks present and functional to me.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/34077605/212339700-50d465be-f878-4b93-b29e-9d0b7b2fdb83.png">
<img width="352" alt="image" src="https://user-images.githubusercontent.com/34077605/212339741-ccc6612c-ee58-4fff-ae07-a06353090899.png">

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
